### ISSUE TITLE
[nss] Failed to build

### ISSUE THREAD

**Issue by uberhacker** [skycocker/chromebrew#4631]
```
Unpacking archive using 'tar', this may take a while...
Building from source, this may take a while...
Rename all *.la files to *.la_tmp
nss failed to build: `CPPFLAGS='-I/usr/local/include/nspr' ./build.sh --opt --gcc --system-nspr --system-sqlite --disable-tests` exited with 1
```

**Comment by satmandu** [comment]
This requires a fixed version of python27. Testing a working version now.

**Comment by uberhacker** [comment]
@satmandu: Even after the new python27 upgrade, I still cannot compile.

**Comment by satmandu** [comment]
What machine are you testing on? Let me spin up a docker image for that machine and try?

**Comment by uberhacker** [comment]
It works on i686 but no other architecture.  armv7l and x86_64 both fail right from the get go.

**Comment by satmandu** [comment]
The quick fail sounds like it is still a python issue. I'm spinning up a armv8l docker instance to see what happens.

**Comment by satmandu** [comment]
Compiled for me on the armv8l docker image, built from fievel's M86 recovery image.

**Comment by satmandu** [comment]
Here's the docker script I used today on my RPI4b: https://gist.github.com/satmandu/791dd6e982d365dda6bbba5c332f0f73
Make a folder, drop the new python3.rb into that folder, and run that script thus:
```chromeos_docker https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_13421.99.0_veyron-fievel_recovery_stable-channel_fievel-mp.bin.zip fievel 86```
That dropped me to a prompt after about 15 min, where I was able to run ```crew install nss```.

```
chronos@m86armv7l ~ $ crew const
ARCH=armv7l
ARCH_LIB=lib
CHROMEOS_RELEASE=86
CREW_BREW_DIR=/usr/local/tmp/crew/
CREW_BUILD=armv7l-cros-linux-gnueabihf
CREW_CMAKE_LIBSUFFIX_OPTIONS=-DCMAKE_INSTALL_PREFIX=/usr/local -DCMAKE_LIBRARY_PATH=/usr/local/lib -DLIB_SUFFIX= -DCMAKE_BUILD_TYPE=Release --build=armv7l-cros-linux-gnueabihf --host=armv7l-cros-linux-gnueabihf --target=armv7l-cros-linux-gnueabihf
CREW_CMAKE_OPTIONS=-DCMAKE_INSTALL_PREFIX=/usr/local -DCMAKE_LIBRARY_PATH=/usr/local/lib -DCMAKE_BUILD_TYPE=Release --build=armv7l-cros-linux-gnueabihf --host=armv7l-cros-linux-gnueabihf --target=armv7l-cros-linux-gnueabihf
CREW_CONFIG_PATH=/usr/local/etc/crew/
CREW_DEST_DIR=/usr/local/tmp/crew/dest
CREW_DEST_HOME=/usr/local/tmp/crew/dest/home/chronos/user
CREW_DEST_LIB_PREFIX=/usr/local/tmp/crew/dest/usr/local/lib
CREW_DEST_MAN_PREFIX=/usr/local/tmp/crew/dest/usr/local/share/man
CREW_DEST_PREFIX=/usr/local/tmp/crew/dest/usr/local
CREW_LIB_PATH=/usr/local/lib/crew/
CREW_LIB_PREFIX=/usr/local/lib
CREW_LIB_SUFFIX=
CREW_MAN_PREFIX=/usr/local/share/man
CREW_MESON_OPTIONS=-Dprefix=/usr/local -Dlibdir=/usr/local/lib -Dmandir=/usr/local/share/man -Dbuildtype=release
CREW_NOT_COMPRESS=
CREW_NOT_STRIP=
CREW_NPROC=4
CREW_OPTIONS=--prefix=/usr/local --libdir=/usr/local/lib --mandir=/usr/local/share/man --build=armv7l-cros-linux-gnueabihf --host=armv7l-cros-linux-gnueabihf --target=armv7l-cros-linux-gnueabihf
CREW_PREFIX=/usr/local
CREW_VERSION=1.5.1
HOME=/home/chronos/user
LIBC_VERSION=2.27
USER=chronos
```

**Comment by uberhacker** [comment]
Fixed in #4770.
